### ISSUE TITLE
russian Text-To-Speech

### ISSUE THREAD

**Issue by olibeniron** [BigWigsMods/BigWigs_Voice#19]
Good day.
Is it possible to switch english Text-To-Speech into russian Text-To-Speech?
It does not a problem for me, but apparently many my guildmembers want to use this function, but they are like total zero in English.
I found several files in the main folder, but they translate only the core of the addon, not Text-To-Speech.

**Comment by sehra** [comment]
I would have to create a separate addon with Russian translations since this addon only support English. I can take a look how much work it would be.

**Comment by olibeniron** [comment]
I can do all russian translations if it is necessary. Though I won't be able to help with programming or moderating addon from inside.

**Comment by sehra** [comment]
I created a project on Curse for a Russian version. It should come online as soon as the manual approval is done.

**Comment by sehra** [comment]
You can give [BigWigs Voice (RU)](https://wow.curseforge.com/projects/bigwigs_voice_ru) a try. Still a work in progress but most of the spells from Battle for Azeroth is present.

Just make sure to only have one of the voice addons active, there is no way to switch between them.

**Comment by olibeniron** [comment]
Thanks. I cheked it. It doens't work. 

I tried russian-text/russian-voices combination and russian-text/english-voices combination (game settings) with "BigWigs Voice (RU)" on. Text-To-Speech function was on. I tested it in two dung - Temple and Manor. Addon used common bigwigs sounds, like Alarm, Info, ect. 

I switched back to English, it means I disabled Voice-Ru and enabled Voice-Eng. It worked ONLY with russian-text/english-voices combination (game settings), which was, well, logical! It didn't work with russian-text/russian-voices.
